src/components/TodoHeader/TodoHeader.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FormEvent, useState } from 'react';
+import cn from 'classnames';
+import { ErrorMessages } from '../../types/ErrorMessages';
+
+type Props = {
+  onError: (value: ErrorMessages | null) => void;
+  createTodo: (title: string) => void;
+  isInputDisabled: boolean;
+};
+
+export const TodoHeader: React.FC<Props> = ({
+  onError,
+  createTodo,
+  isInputDisabled,
+}) => {
+  const [newTodo, setNewTodo] = useState<string>('');
```

Comment:
isn't this one a `newTodoTitle`?
A bit tricky one here, by its name I expect it to be an object